typescript/src/schema/api.ts

```diff
@@ -93,6 +93,11 @@ export class API {
         this.port = port ?? this.port ?? '443';
         serviceNamesList.push(service.name || this.naming.name);
       });
+    if (serviceNamesList.length === 0) {
+      throw new Error(
+        `Can't find ${this.naming.name}'s service names, please make sure that service names are defined in the proto file.`
```

Comment:
The error message looks good! Let's maybe say `...make sure that services are defined...`, not the service names.

Reply:
will do